src/app/tests/TestReadInteraction.cpp

```diff
@@ -143,79 +246,10 @@ void TestReadInteraction::GenerateReportData(nlTestSuite * apSuite, void * apCon
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 }
 
-void TestReadInteraction::TestReadClient(nlTestSuite * apSuite, void * apContext)
```

Comment:
Why remove this?

Reply:
Accidental, remove, just bring it back, thanks